aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -197,6 +197,11 @@ module aptos_framework::stake {
         validators: vector<IndividualValidatorPerformance>,
     }
 
+    // The beneficiary address for the operator.
+    struct Beneficiary has key {
```

Comment:
Create a resource group for this so any more resources added to the stake pool in the future can be in the same resource group for storage efficiency

Reply:
This has been removed from the `stake` module.